Controller/Cart/Hints.php

```diff
@@ -20,7 +20,7 @@
 use Magento\Framework\App\Action\Action;
 use Magento\Framework\App\Action\Context;
 use Bolt\Boltpay\Helper\Bugsnag;
-use Magento\Framework\Exception\LocalizedException;
+use Exception;
```

Comment:
Learning question: Why change the exception used by the class as opposed to changing the exception thrown to be a `LocalizedException`?

Reply:
@njfaries I removed it (Magento\Framework\Exception\LocalizedException) since it's not used anywhere in the class Controller/Cart/Hints.php